src/components/Checkout/OrderTotals.vue

```diff
@@ -446,8 +446,6 @@ export default {
 	}
 
 	.upc-disclaimer {
-		display: flex;
-		justify-content: right;
```

Comment:
Do we need to revert these changes? What does this copy look like now on desktop?

Reply:
reverted, done